### PR TITLE
Array helpers and errors

### PR DESCRIPTION
**Pull request by wallymathieu**
Add errors method on arrayhelpers type in order to be able to extract errors per row

### CODE REVIEW

File: src/types.props.ts [hunk 1]
```diff
@@ -59,10 +59,7 @@ export type FormikReimaginedProps<Values> = FormikReimaginedState<Values> &
 /**
  * Render properties of field array. Accessible through render and children.
  **/
-export interface FieldArrayRProps<Value> extends ArrayHelpers<Value> {
```
Comment (wallymathieu) [inline src/types.props.ts]: In order to help typescript with type inference, let's remove one level of ....

Comment (davidwincent) [reply src/types.props.ts]: looks good, does this break compatability?

Comment (wallymathieu) [reply src/types.props.ts]: Yes, though we haven't made a 1.0.0 release yet. But I can add a deprecated alias if you want.

Comment (wallymathieu) [reply src/types.props.ts]: https://github.com/jaredpalmer/formik/pull/2237

**Pull request merged**